src/main/java/net/mcreator/ui/action/BasicAction.java

```diff
@@ -18,17 +18,25 @@
 
 package net.mcreator.ui.action;
 
+import net.mcreator.ui.action.accelerators.Accelerator;
+
 import javax.swing.*;
 import java.awt.event.ActionListener;
 
 public class BasicAction extends UnregisteredAction {
 
 	protected ActionRegistry actionRegistry;
+	protected Accelerator accelerator;

```

Comment:
This, argument, getter and subclasses should mark this with @Nullable to be more clear that this can be set to null safely